src/main/java/edu/cornell/kfs/concur/service/ConcurRequestExtractFileValidationService.java

```diff
@@ -0,0 +1,5 @@
+package edu.cornell.kfs.concur.service;
+
+public interface ConcurRequestExtractFileValidationService {
+	
```

Comment:
What methods are expected in this validation service? I think it would be helpful to define this interface even if you don't add the implementation for the methods at this point.